test/user.test.js

```diff
@@ -1714,6 +1714,30 @@ describe('User', function() {
               .to.equal('#/some-path?a=1&b=2');
           });
       });
+
+      it('build additional confirmation url and pass on template', function() {
+        return User.create({email: 'test1@example.com', password: 'pass'})
+          .then(user => {
+            let newVerifyUrl;
+            return user.verify({
+              type: 'email',
+              to: user.email,
+              from: 'noreply@myapp.org',
+              redirect: '#/some-path?a=1&b=2',
+              templateFn: (options, cb) => {
+                options.verifyMovil = '/redirect.html?uid=' + user.id + ';token=' + options.verificationToken;
```

Comment:
I would prefer to simplify this test - the purpose is to verify that `options.templateFn` receives `options.verificationToken`.

See the test above for an example.

```js
// ...
templateFn: (options, cb) => {
  actualVerificationToken = options.actualVerificationToken;
  cb(null, 'dummy body');
},
// ...
.then(token => {
  expect(token).to.exist();
});
```